.github/workflows/cherry-pick.yaml

```diff
@@ -0,0 +1,26 @@
+on:
+  pull_request:
+  workflow_dispatch:
+    inputs:
+      source_revision:
+        description: "Revision to cherry-pick"
+        required: true
+      target_branch:
+        description: "Target branch: i.e. aptos-release-v1.14"
+        required: true
+
+jobs:
+  # TODO: determine branch from PR label
+  cherry_pick_pr:
+    runs-on: ubuntu-latest
+    name: Cherry pick into ${{ inputs.target_branch || 'aptos-release-v1.14' }}
```

Comment:
Perhaps it would be better to choose a nonexistent branch as default, to ensure that one can't pick `aptos-release-v1.14` by accident.

Reply:
this is for testing, marking as draft